src/app.ts

```diff
@@ -64,8 +64,10 @@ export default class App {
     app.set('layout', 'layouts/layout.ejs');
 
     app.use(express.static(path.join(__dirname)));
-
-    app.get('/', (_req, res) => res.render('pages/index.ejs'));
+    const inspectletKey = ConfigService.hasValue('inspectlet.wid') ? ConfigService.getStringValue('inspectlet.wid') : '';
+    app.get('/', (_req, res) => {
+      res.render('pages/index.ejs', { inspectlet_key: inspectletKey || '' });
```

Comment:
Nit: Can remove the additional `|| ''` from here. It has been already processed on line above.